airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -84,12 +85,16 @@
 import java.util.UUID;
 import java.util.function.Consumer;
 import java.util.function.Function;
+import javax.annotation.Nullable;
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
 public class ConnectionManagerWorkflowImpl implements ConnectionManagerWorkflow {
 
+  private static final String GENERATE_CHECK_INPUT_TAG = "generate_check_input";
+  private static final int GENERATE_CHECK_INPUT_CURRENT_VERSION = 1;
```

Comment:
... time to version I guess!

Reply:
If you didn't version, I think that running syncs would fail, but perhaps only before the sync stage, not during or afterwords?  If that's the case, and they retry, that's not so bad?

Reply:
Yeah, that sounds right. Avoiding the failure wasn't too bad here with the version though so seemed worth it to keep history compatibility. FWIW there's also a test that verifies history can be replayed that not versioning would break 😄